infomaniak/kDrive.cmake

```diff
@@ -35,12 +35,14 @@ if("$ENV{KDRIVE_DEBUG}" STREQUAL "1")
     set( KDRIVE_API_V2_URL "https://api.kdrive.preprod.dev.infomaniak.ch/2" )
     set( NOTIFY_KDRIVE_V2_URL "https://notify.kdrive.preprod.dev.infomaniak.ch/2" )
     set( LOGIN_URL "https://login.preprod.dev.infomaniak.ch" )
+    set( INFOMANIAK_API_URL "https://api.infomaniak.com/1" )
```

Comment:
If the two `set` statements are really the same, you can put this common statement outside of the `if-else` blocks.